src/components/DateRangeSelect/PDateRangeSelectRelative.vue

```diff
@@ -87,4 +114,15 @@
 .p-date-range-select-relative {
   min-width: 300px;
 }
+
+.p-date-range-select-relative .p-select-option__text { @apply
+  w-full
+}
```

Comment:
PSelectOption doesn't allow for full width content of its slot. This seemed like the simplest and least risky option for getting the layout necessary without having to refactor PSelectOption and test everything that uses it.